client/src/components/Panels/Common/ToolSearch.vue

```diff
@@ -36,12 +40,73 @@
                     <icon icon="redo" />
                     <span>{{ "Cancel" | localize }}</span>
                 </b-button>
+                <b-button title="Search Help" size="sm" @click="showHelp = true">
+                    <icon icon="question" />
+                </b-button>
+                <b-modal v-model="showHelp" title="Tool Advanced Search Help" ok-only>
+                    <div v-html="helpHtml"></div>
+                </b-modal>
             </div>
         </div>
     </div>
 </template>
 
 <script>
+const helpHtml = `<div>
```

Comment:
Why is this not part of the template?

Reply:
Oh we were doing something similar in `WorkflowList` so I did it here too. I can move it to the template

Reply:
I think the best place to put this would be it's own component, since it is quite large. Inline is fine too.
Including it as strings we miss out on editor and language server features, and increase the potential for bugs

Reply:
@ahmedhamidawan Please don't refactor that one until the grids PRs are merged :) (e.g. https://github.com/galaxyproject/galaxy/pull/16045)